noxfile.py

```diff
@@ -14,3 +14,4 @@
 
 # Sets the default session to `--list`
 nox.options.sessions = []
+nox.options.reuse_existing_virtualenvs = True
```

Comment:
without reusing the virtual envs the runs take much longer, and the first run will also take much longer as it installs and caches it.

This still catches whenever a package version changes, so no downsides here